### PR TITLE
Add support for Software Component Relations

### PR DESCRIPTION
**Pull request by hottek**
This PR contains the abapGit handler to support the object type SWCR.
This object type is already maintained in the AFF https://github.com/SAP/abap-file-formats/pull/539.

### CODE REVIEW

File: src/objects/aff/zcl_abapgit_aff_registry.clas.abap [hunk 1]
```diff
@@ -46,21 +46,22 @@ CLASS ZCL_ABAPGIT_AFF_REGISTRY IMPLEMENTATION.
...
+    register( 'DSFI' ).
+    register( 'DRAS' ).
     register( 'EVTB' ).
     register( 'EEEC' ).
     register( 'GSMP' ).
     register( iv_obj_type     = 'INTF'
               iv_experimental = abap_true ).
     register( 'SMBC' ).
+    register( 'SWCR' ).
     register( 'NONT' ).
```
Comment (larshp) [inline src/objects/aff/zcl_abapgit_aff_registry.clas.abap:65]: only expect SWCR to be added here, no changes to other stuff

**Comment by larshp** [comment]
can you help opening a PR in https://github.com/abapGit-tests/SWCR with an example file?

File: src/objects/aff/zcl_abapgit_aff_registry.clas.abap (hunk 1 above)
Comment (hottek) [reply src/objects/aff/zcl_abapgit_aff_registry.clas.abap:65]: reverted the changes already

**Comment by hottek** [comment]
@larshp yes, will do.

**Comment by hottek** [comment]
Added an example with https://github.com/abapGit-tests/SWCR/pull/1.

**Comment by larshp** [comment]
welcome to abapGit as a contributor 🎉

**Pull request merged**